cmd/kubeops/internal/handler/handler_test.go

```diff
@@ -140,6 +214,8 @@ func TestActions(t *testing.T) {
 			switch test.Action {
 			case "create":
 				CreateRelease(*cfg, response, req, test.Params)
+			case "delete":
+				DeleteRelease(*cfg, response, req, test.Params)
```

Comment:
See what you think, but in https://github.com/kubeapps/kubeapps/pull/1439 I've used a separate test case for Rollback, rather than adding to this one. As mentioned there, I know it's probably just following the precedent from tiller-proxy, but it means tests for the individual handlers can't be focused. For example, here we've had to add a `t.SkipNow()` which is only relevant for a certain Delete action. Anyway, happy for this to land as is, just pointing out in case you choose to separate them. I can re-base my diff once yours lands.

Reply:
Smaller functions are always better.
To avoid overhead complications with multiple test PR's coming in right now, and the eventuality of merge conflicts. I plan to postpone splitting up the `TestActions` function to smaller ones, once the test PR's have landed. If that's okay with you.

Reply:
Yep, that's fine. Let's land this one (and your other one), then tomorrow I'll rebase my `Rollback` one with the separate test and we can go from there.